<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSplitClass</class>
 <widget class="QWidget" name="DataSplitClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>一体化控制单元-数据拆包</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DataSplit.qrc">
    <normaloff>:/DataSplit/image_1/logo64.ico</normaloff>:/DataSplit/image_1/logo64.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>201</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>接收串口</string>
     </property>
     <widget class="QPushButton" name="Rbt_OpenCom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover{
 	background-color:transparent;
 	border-image: url(:/DataSplit/image_1/bt_hover.png);
    border:0px;
	color: rgb(255, 255, 255);}
QPushButton:pressed{
 	background-color:transparent; 
 	border-image: url(:/DataSplit/image_1/bt_active.png);
    border:0px;
	color: rgb(255, 255, 255);}
QPushButton {
  	background-color:transparent;
  	border-image: url(:/DataSplit/image_1/bt_normal.png);
    border:0px;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>串口号：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Rcb_com">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid darkgray;</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="Rcb_Baudrate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid darkgray;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>波特率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="Rlb_state">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>151</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/DataSplit/image_1/Connection statuP_red.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Rlb_stateT">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>状态：未打开</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Rcb_saveFile">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>保存接收数据</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Rcb_showRData">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>显示接收数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_clear">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清空显示</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>接收数据</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTextEdit" name="Log_textEdit">
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="accessibleDescription">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(250, 250, 250);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>已发数据</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTextEdit" name="Log_textEdit_2">
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="accessibleDescription">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(250, 250, 250);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>201</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>发送串口</string>
     </property>
     <widget class="QPushButton" name="Sbt_OpenCom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover{
 	background-color:transparent;
 	border-image: url(:/DataSplit/image_1/bt_hover.png);
    border:0px;
	color: rgb(255, 255, 255);}
QPushButton:pressed{
 	background-color:transparent; 
 	border-image: url(:/DataSplit/image_1/bt_active.png);
    border:0px;
	color: rgb(255, 255, 255);}
QPushButton {
  	background-color:transparent;
  	border-image: url(:/DataSplit/image_1/bt_normal.png);
    border:0px;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>串口号：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Scb_com">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid darkgray;</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="Scb_Baudrate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid darkgray;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>波特率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="Slb_state">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>151</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/DataSplit/image_1/Connection statuP_red.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Slb_stateT">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>状态：未打开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Sbt_sendData">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>160</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover{
 	background-color:transparent;
 	border-image: url(:/DataSplit/image_1/bt_hover.png);
    border:0px;
	color: rgb(255, 255, 255);}
QPushButton:pressed{
 	background-color:transparent; 
 	border-image: url(:/DataSplit/image_1/bt_active.png);
    border:0px;
	color: rgb(255, 255, 255);}
QPushButton {
  	background-color:transparent;
  	border-image: url(:/DataSplit/image_1/bt_normal.png);
    border:0px;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>发送文件</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Scb_showRData">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>显示发送数据</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DataSplit.qrc"/>
 </resources>
 <connections/>
</ui>
